openreview/journal/journal.py

```diff
@@ -17,14 +17,15 @@
 
 class Journal(object):
 
-    def __init__(self, client, venue_id, secret_key, contact_info, full_name, short_name):
+    def __init__(self, client, venue_id, secret_key, contact_info, full_name, short_name, website):
```

Comment:
set `website=None` here

Reply:
Done. Reverted the TMLR test as well to pass no website

Reply:
thanks!